pkgs/os-specific/linux/roccat-tools/default.nix

```diff
@@ -0,0 +1,130 @@
+	    substituteInPlace $out/share/applications/roccatryosmkconfig.desktop --replace "Exec=" "Exec=su -c "
+	    substituteInPlace $out/share/applications/roccattyonconfig.desktop --replace "Exec=" "Exec=su -c "
+	    substituteInPlace $out/share/applications/roccatarvoconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkoneconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkonepuremilitaryconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkonextdopticalconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatnythconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatryostklconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatiskuconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkoneplusconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkonepureopticalconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkovaplusconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatpyraconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatsavuconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatiskufxconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkonepureconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatkonextdconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatluaconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccatryosmkconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    substituteInPlace $out/share/applications/roccattyonconfig.desktop --replace "Terminal=false" "Terminal=true"
+	    touch --no-create /usr/share/icons/hicolor
+	    # gtk-update-icon-cache -q /usr/share/icons/hicolor
+	    # Don't need to do the below because for now we run the
+	    # application using pkexec
+	    # groupadd --force --system roccat
+	    # mkdir --parents /var/lib/roccat
+	    # chown root:roccat /var/lib/roccat
+	    # chmod 2770 /var/lib/roccat
+	    # udevadm control --reload-rules
+	}
+
+        genericBuild
+
+      )
+    '';
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "Roccat hardware support for Linux";
```

Comment:
@jagajaga Uhm, if we remove "Roccat" from the description, it won't make much sense anymore. But maybe something like `Various tools for Roccat hardware` is more descriptive..